ArduPlane/GCS_Mavlink.cpp

```diff
@@ -378,6 +378,21 @@ void Plane::send_pid_tuning(mavlink_channel_t chan)
             return;
         }
     }
+    if (g.gcs_pid_mask & 0x10 && flight_stage == AP_Vehicle::FixedWing::FLIGHT_LAND) {
```

Comment:
the jump from 8 to 0x10 is tricky. It would be more clear if it was:
(1 << PID_TUNING_ROLL)
(1 << PID_TUNING_PITCH)
...
(1 << PID_TUNING_LANDING)